lib/process-node-definitions.js

```diff
@@ -47,6 +49,8 @@ var ProcessNodeDefinitions = function(React) {
             });
         }
 
+        elementProps.key = ++index;
```

Comment:
Hey @lithin, since the `key` property is always set, do you mind moving this up to where `elementProps` is defined? So that it will look like this:
```
var elementProps = {
    key: ++index
};
```

Thanks,

Mike